cmd/logs/export/runner.go

```diff
@@ -0,0 +1,63 @@
+package export
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kind/pkg/cluster"
+)
+
+type runner struct {
+	flag   *flag
+	logger micrologger.Logger
+	stdout io.Writer
+	stderr io.Writer
+}
+
+func (r *runner) Run(cmd *cobra.Command, args []string) error {
+	ctx := context.Background()
+
+	err := r.flag.Validate()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	err = r.run(ctx, cmd, args)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
+
+func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) error {
+	var err error
+
+	var known bool
```

Comment:
Why define the variable here?
`known, err := cluster.IsKnown(r.flag.Name)` should do the job imo

Reply:
It was copied from the other commands. But yes that is simpler. Done.